src/main/resources/application.yaml

```diff
@@ -83,6 +86,14 @@ idam:
     redirect_uri: https://localhost:9000/oauth2/callback
     secret: OOOOOOOOOOOOOOOO
 
+document_management:
+  url: http://localhost:4506a
+
+docmosis:
+  tornado:
+    url: "http://localhost:5433"
```

Comment:
we were not using quotes in config file, maybe we should stick to that so it's consistent